template/package.json

```diff
@@ -9,7 +9,7 @@
     "fmt": "mongodb-js-fmt ./*.js ./test/*.js",
     "check": "mongodb-js-precommit './src/**/*{.js,.jsx}' './test/**/*.js' index.js",
     "compile": "npm run jsx && npm run less",
-    "jsx": "babel ./src --out-dir ./lib --presets react",
+    "jsx": "babel --presets react --plugins transform-class-properties --out-dir ./lib ./src",
```

Comment:
Does babel not use the babelrc file by default? Is there an argument to set the config file to use? It'd be nice to not repeat configuration.